spec/features/manage_cmr/subscriptions/subscription_form_validation_spec.rb

```diff
@@ -113,6 +113,32 @@
           expect(page).to have_no_content('Collection Concept ID is required.')
         end
       end
+
+      context 'when submitting a valid query' do
+        # TODO: might want to randomly generate a query based off of the
+        # allowed parameters and allowed special characters for this test
+        let(:valid_query) { '?bounding_box=-10,-5,10,5&Platform[]=1B&platform[]=NASA GLOBAL HAWK 872 AIRCRAFT&version=1&attribute\[\]=string,MISSION_NAME,Big Island\, HI&created_at[]=2015-01-01T10:00:00Z' }
+
+        before do
+          # need to input the query before filling in other fields because otherwise
+          # clicking the submit button doesn't actually submit but only causes a blur
+          fill_in 'Query', with: valid_query
+          fill_in 'Name', with: name
+          fill_in 'Collection Concept ID', with: collection_concept_id
+
+          within '.subscription-form' do
+            click_on 'Submit'
+          end
+        end
+
+        it 'displays the proper validation errors withing the form' do
```

Comment:
within

Reply:
hah, that typo is literally copy/pasted through the whole file. If I have to re-run the build, I will make the change, otherwise I will leave it for another time.